packages/playground/src/dashboard/dao_view.vue

```diff
@@ -218,9 +218,11 @@
                 >Votes are weighted based on the farmers stake in the network. One vote by default is 1 weight.</span
               >
               <span> If the farmers has nodes, the weight of the vote is calulcated as following:</span>
-              <ul>
-                <li>Sum of all nodes of the farmer: (node CU * 2 + node SU)</li>
-              </ul>
+              <span
+                ><b>
+                  The farmer's vote weight is equal to the sum of all nodes of the farmer: (node CU * 2 + node SU)</b
+                ></span
```

Comment:
maybe "The farmer's vote weight is the sum of all the farmer's nodes weight. A node's weight is calculated as: node CU * 2 + node SU." ?

Reply:
updated in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1551/commits/e69ecc7008aa43ebd3075f4b56074fbd1b1ddce0